ydb/library/yql/dq/actors/compute/dq_compute_memory_quota.h

```diff
@@ -55,6 +56,23 @@ namespace NYql::NDq {
             }
         }
 
+        // This callback is created for testing purposes and will be enabled only with spilling.
+        // Most likely this callback will be removed after KIKIMR-21481.
+        void TrySetIncreaseMemoryLimitCallbackWithRSSControl(NKikimr::NMiniKQL::TScopedAlloc* alloc) {
+            if (!CanAllocateExtraMemory) return;
+            const ui64 limitRSS = std::numeric_limits<ui64>::max();
```

Comment:
Why do we use max ui64?

Reply:
This value is expected to be changed manually while testing. Here I used max just as a placeholder